packages/firebase/index.d.ts

```diff
@@ -7743,6 +7743,15 @@ declare namespace firebase.firestore {
      * @webonly
      */
     experimentalForceLongPolling?: boolean;
+
+    /**
+     * Whether to skip nested properties that are set to `undefined` during
+     * object serialization. If set to `true`, these properties will be skipped
```

Comment:
Very much a nit, but we prefer present over future when possible.  Here it cold be "properties are skipped . . . . the SDK throws . . ."

Reply:
Updated. Thanks for the review!